game/src/engine/physics/HelperUtils.h

```diff
@@ -8,4 +8,24 @@ struct Command
     physx::PxF32 throttle;
     physx::PxF32 steer;
     physx::PxF32 duration;
+
+    Command& operator+(const Command& a)
+    {
+        brake = brake + a.brake;
+        throttle = throttle + a.throttle;
+        steer = steer + a.steer;
+
+        // not adding the duration as it is not required.
+        return *this;
+    }
+
+    Command& operator+=(const Command& a)
+    {
+        brake += a.brake;
+        throttle += a.throttle;
+        steer += a.steer;
+
+        // not adding the duration as it is not required.
+        return *this;
+    }
```

Comment:
Can we rename this file to something like `VehicleCommand` instead of `HelperUtils`? I find that eventually files with names like `Utils` almost always end up becoming massive and containing random things :P